### PR TITLE
Add Trusted Types symbols

### PR DESCRIPTION
**Pull request by vrana**

### CODE REVIEW

**Comment by vrana** [comment]
As suggested by @rkirov.

File: src/main/java/com/google/javascript/clutz/PlatformSymbols.java [hunk 1]
```diff
@@ -1458,6 +1458,13 @@
           "TransitionEventInit",
           "Transport",
           "TreeWalker",
+          "TrustedHTML",
```
Comment (rkirov) [inline src/main/java/com/google/javascript/clutz/PlatformSymbols.java]: Add some comments around these as they don't trully match the original comment at the top.

Comment (vrana) [reply src/main/java/com/google/javascript/clutz/PlatformSymbols.java]: Done.

**Comment by koto** [comment]
Is it possible to rerun the CircleCI tests and merge the PR? The failures don't seem related.

File: src/main/java/com/google/javascript/clutz/PlatformSymbols.java [hunk 2]
```diff
@@ -1458,13 +1458,16 @@
           "TransitionEventInit",
           "Transport",
           "TreeWalker",
+
+          // https://github.com/WICG/trusted-types
           "TrustedHTML",
```
Comment (rkirov) [inline src/main/java/com/google/javascript/clutz/PlatformSymbols.java]: This comment is insufficient for maintainers of clutz to know what's going on. Through conversation with 	
@koto I got a bit more of a context. Please add something of this sort:

```
The TrustedTypes (TT) browser api is experimental, but there is an implementation in Chrome. Closure's platform externs will be made to unconditionally depend on TT types, and similarly all TS compilations will depend on https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/trusted-types/index.d.ts

In that sense, TT can be seen by clutz as a platform API (albeit experimental). By adding it to this list we guarantee that the symbols are emitted without the clutz prefix and transparently mapped between the hand-written externs and .d.ts.

For more info see https://github.com/WICG/trusted-types
```

**Comment by rkirov** [comment]
Rebased and merged with https://github.com/angular/clutz/pull/855

**Pull request closed without merging**